schedule/functional/extra_tests_gnome.yaml

```
@@ -0,0 +1,16 @@
+name:   extra_tests_gnome
+description:    >
+    Maintainer: asmorodskyi, okurz.
+    Extra tests which were designed to run on gnome >
```

Sorry if I confused you. The '>' is only needed after `description` field.

```suggestion
    Extra tests from desktop applications module which were designed to run on gnome
```